std_msgs/String color
geometry_msgs/Pose pose